src/session/sidebar/row.rs

```diff
@@ -139,11 +139,14 @@ impl Row {
                 self_.bottom_box.set_visible(true);
 
                 // Chat properties expressions
-                let title_expression = gtk::PropertyExpression::new(
-                    Chat::static_type(),
-                    gtk::NONE_EXPRESSION,
-                    "title",
-                );
+                let mut title_expression = Chat::title_expression(&chat);
```

Comment:
This is a bit messy (it was already a bit messy before). I pushed #196 that will simplify things quite a bit, I suggest rebasing the branch on that and I suggest rethinking how all of this works now.

The `Chat::title_expression()` is an expression of the `Chat::title` property. This is not what we want: we want an expression that given a `Chat` it gives us a display name (so not necessary the `Chat::title`). So, I suggest writing a `Chat::display_name_expression()` that does the following:
```rust
if let Some(user) = chat.type_().user() {
    user.display_name_expression()
} else {
   // Return `Chat::title` property expression
}
```

Then...